src/types/index.ts

```diff
@@ -0,0 +1,20 @@
+import { IntegrationStepIterationState } from '@jupiterone/jupiter-managed-integration-sdk';
+
+export * from './entities';
+export * from './relationships';
+
+export interface OktaIntegrationConfig {
+  oktaApiKey: string;
+  oktaOrgUrl: string;
+}
+
+export interface OktaIntegrationStepIterationState
+  extends IntegrationStepIterationState {
+  state: {
+    after?: string;
+    seen?: number;
+    limit?: number;
+    pages?: number;
+    applicationIndex?: number;
+  };
+}
```

Comment:
This type should not be used, it was part of tracking across iterative invocations of the Lambda resource fetch functions.

Reply:
Removed